src/jdk.jpackage/share/classes/jdk/jpackage/internal/OverridableResource.java

```diff
@@ -231,6 +231,10 @@ private Source sendToConsumer(ResourceConsumer consumer) throws IOException {
                 return source.getKey();
             }
         }
+
+        Log.verbose(I18N.format("message.no-default-resource",
+                publicName, getPrintableCategory(), publicName));
+
```

Comment:
I still believe this log message is misleading.

Reply:
`[18:34:04.625] No default package resource postinstall [PKG postinstall script] (add postinstall to the resource-dir to customize).`
`[18:34:04.625] No default package resource preinstall [PKG preinstall script] (add preinstall to the resource-dir to customize).
`
This is example of log message. Do you have any suggestions on how to improve it?

Reply:
This change adds the following log message:
```
[22:09:41.559] No default package resource null [script to run after application image is populated] (add null to the resource-dir to customize).
```
I can see it in PkgScriptsTest and in SimplePackageTest.

Reply:
> Do you have any suggestions on how to improve it?

No, I don't know off the top of my head. This is a broader issue: jpackage should write all customizable resources in the log.

Reply:
Let's address customizable resource logging in a separate patch. I filed [JDK-8369334](https://bugs.openjdk.org/browse/JDK-8369334) for this effort.

Reply:
I just fix it. Let me know what you think. I can revert my last commit.